lib/galaxy/config/schemas/config_schema.yml

```diff
@@ -3764,14 +3764,14 @@ mapping:
           Activate this only if you have setup a Celery worker for Galaxy and you have
           configured the `celery_conf` option below. Specifically, you need to set the
           `result_backend` option in the `celery_conf` option to a valid Celery result
-          backend URL.
+          backend URL. By default, Galaxy uses an SQLite database at '<data_dir>/results.sqlite' for storing task results.
+          Please use a more robust backend (e.g. Redis) for production setups.
           For details, see https://docs.galaxyproject.org/en/master/admin/production.html#use-celery-for-asynchronous-tasks
           
       celery_conf:
         type: any
         required: false
         default:
-          result_backend: redis://127.0.0.1:6379/0
```

Comment:
Would:

```yaml
         result_backend:
```

work as a default? That would make it clearer where the option goes.

Reply:
I can try that, maybe we can also add `broker_url` while we are at it...

Reply:
I like it! we need to set it to `null explicitly` so it is valid YAML I think.